AlphaWallet/Settings/ViewModels/SettingsViewModel.swift

```diff
@@ -203,3 +375,22 @@ enum SettingsSection {
         }
     }
 }
+
+extension SettingsSection: Hashable {
+    static func == (lhs: SettingsSection, rhs: SettingsSection) -> Bool {
```

Comment:
Think this is not necessary and can be auto-synthesized. (also the `default:` can introduce bugs when we add new cases and the compiler can't remind us to update the `==` function). Probably the other `==` function(s) too?